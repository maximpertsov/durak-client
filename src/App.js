import 'semantic-ui-css/semantic.min.css';
import 'App.css';

import React from 'react';

import Game from 'scenes/Game';
import Home from 'scenes/Home';

const App = () => (
  <div className="App">
    <Home />
    <Game />
  </div>
);

export default App;
