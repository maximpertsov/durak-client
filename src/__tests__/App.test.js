import React from 'react';

import App from 'App';
import { shallow } from 'enzyme';

test('snapshot', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
