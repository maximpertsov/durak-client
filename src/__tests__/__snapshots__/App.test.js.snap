// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="App"
>
  <Home />
  <Game />
</div>
`;
